packages/next/src/server-build.ts

```diff
@@ -1149,7 +1149,7 @@ export async function serverBuild({
     }
   }
 
-  const rscHeader = routesManifest.rsc?.header?.toLowerCase() || '__rsc__';
+  const rscHeader = routesManifest.rsc?.header?.toLowerCase() || 'rsc';
```

Comment:
```suggestion
  const rscHeader = routesManifest.rsc?.header?.toLowerCase() || '__rsc__';
```
Shouldn't this still be `__rsc__` for backwards compat as the new value is read from the manifest?